### ISSUE TITLE
Laravel 10 Support

### ISSUE THREAD

**Issue by gogl92** [mostafamaklad/laravel-permission-mongodb#143]
**Is your feature request related to a problem? Please describe.**
To fully support current and future Laravel releases `mongodb/laravel-mongodb` should be implemented.

https://github.com/mongodb/laravel-mongodb

_This package was renamed to mongodb/laravel-mongodb because of a transfer of ownership to MongoDB, Inc. It is compatible with Laravel 10.x. For older versions of Laravel, please refer to the [old versions](https://github.com/mongodb/laravel-mongodb/tree/3.9#laravel-version-compatibility)._ 

**Describe the solution you'd like**
`mongodb/laravel-mongodb` 4 should be required in the composer dependencies instead of `jenssegers/mongodb`

**Describe alternatives you've considered**
I created a fork and tested it with `mongodb/laravel-mongodb` dependency.

**Additional context**
I will open a pull request when ready.

**Comment by zoltansuto** [comment]
@mostafamaklad 
composer changes in itself won't be enough, the namespaces have to be updated as well
https://github.com/mostafamaklad/laravel-permission-mongodb/pull/145
based on work from @webrek 's commits
https://github.com/mostafamaklad/laravel-permission-mongodb/compare/master...webrek:laravel-permission-mongodb:master

**Comment by gogl92** [comment]
> @mostafamaklad 
> composer changes in itself won't be enough, the namespaces have to be updated as well
> https://github.com/mostafamaklad/laravel-permission-mongodb/pull/145
> based on work from @webrek 's commits
> https://github.com/mostafamaklad/laravel-permission-mongodb/compare/master...webrek:laravel-permission-mongodb:master

Right, it'll need the change of the namespaces in the code and the tests, hopefully I can do it this weekend🤞
